code/datums/vote/gamemode.dm

```diff
@@ -24,9 +24,9 @@
 		var/datum/game_mode/M = gamemode_cache[F]
 		if(!M)
 			continue
-		display_choices[F] = capitalize(M.name)
 		additional_text[F] ="<td align = 'center'>[M.required_players]</td>"
 	display_choices["secret"] = "Secret"
+	display_choices["extended"] = "Extended"
```

Comment:
???? Where'd all the options go?

is this based on the version before the revert?

Add the changes here instead:
https://github.com/Skyrat-SS13/Vesta.Bay/blob/faf4b2c9101e0b687b9029b83bd595a747441824/config/config.txt#L413-L415